### PR TITLE
[T] (quests) chaptre_2

### PR DESCRIPTION
**Pull request by nai200**
Co-authored-by: 寒焱炙冰 <realsunmengxiang@foxmail.com>
Co-authored-by: 藏青 <2291848664@qq.com>

### CODE REVIEW

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 1]
```diff
@@ -316,15 +317,15 @@
...
+			title: "工匠青铜"
 			x: -4.75d
 			y: -2.25d
 			dependencies: ["4FF742939987793B"]
 			id: "2AE06ADF534ADDEC"
 			tasks: [{
 				id: "0B0664B538512CF4"
 				type: "item"
-				title: "3x Blocks of Tinker's Bronze"
+				title: "3x 工匠青铜块"
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				title: "3x工匠青铜块"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 2]
```diff
@@ -404,7 +405,7 @@
 				{
 					id: "301991D81A850D16"
 					type: "item"
-					title: "2x Seared Faucets"
+					title: "2x 浇筑口"
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "2x浇筑口"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 3]
```diff
@@ -437,17 +438,17 @@
...
-			title: "Seared Ingot \\& Fuel Gauge"
+			title: "材料量器和燃料量器"
 			x: -7.5d
 			y: -5.5d
 			description: [
-				"Seared Ingot Gauge is ideal for metals. It can hold 32 ingots of fluid (4.6 buckets)."
+				"焦黑材料量器是盛放材料的理想容器，它能够存放32锭的液体（4.6b）"
 				""
-				"Seared Fuel Gauge is basically the same as the Seared Fuel Tank. It can be used as Smeltery heart, and can hold 4 buckets."
+				"从根本上说，焦黑燃料量器和焦黑材料量器相同。它可以作为冶炼炉的心脏，并且能够存放4桶燃料。Seared Fuel Gauge is basically the same as the Seared Fuel Tank. It can be used as Smeltery heart, and can hold 4 buckets."
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"从根本上说，焦黑燃料量器和焦黑材料量器相同。它可以作为冶炼炉的心脏，并且能够存放4桶燃料。"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 4]
```diff
@@ -899,7 +900,7 @@
 			}]
 		}
 		{
-			title: "Flint and _____"
+			title: "打火石和 _____"
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "打火石和_____"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 5]
```diff
@@ -1142,17 +1143,16 @@
...
 		{
-			title: "Scorched Ingot \\& Fuel Gauge"
 			x: -4.75d
 			y: 10.75d
 			description: [
-				"Scorched Ingot Gauge is ideal for metals. It can hold 32 ingots of fluid (4.6 buckets)."
+				"焦褐材料量器是盛放材料的理想容器，它能够存放32锭的液体（4.6b）。"
 				""
-				"Scorched Fuel Gauge is basically the same as the Scorched Fuel Tank. It can be used as Foundry tank, and can hold 4 buckets."
+				"从根本上说，焦褐燃料量器和焦褐材料量器相同，都可以作为焦褐燃料容器。它可以被用作为熔铸炉燃料容器，并且能够存放4桶燃料。Scorched Fuel Gauge is basically the same as the Scorched Fuel Tank. It can be used as Foundry tank, and can hold 4 buckets."
```
Comment (xinyihl) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"从根本上说，焦褐燃料量器和焦褐材料量器相同，都可以作为焦褐燃料容器。它可以被用作为熔铸炉燃料容器，并且能够存放4桶燃料。"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 6]
```diff
@@ -3,13 +3,13 @@
 	group: "0889EAB460A6BD29"
 	order_index: 1
 	filename: "chaptre_2"
-	title: "&bChapter 2&r &6-&r Smeltery and better tools"
+	title: "&b第二章&r&6-&r冶炼厂和工具"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
	title: "&b第二章&r&6-&r冶炼炉和工具"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 7]
```diff
@@ -3,13 +3,13 @@
...
 	icon: "tconstruct:smeltery_controller"
 	default_quest_shape: ""
 	default_hide_dependency_lines: false
 	quests: [
 		{
-			title: "&7Tinkers' Construct Clay Island&r"
+			title: "&7匠魂黏土空岛&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "&7匠魂黏土岛&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 8]
```diff
@@ -18,7 +18,7 @@
 			tasks: [{
 				id: "2BD62C616FCB49F9"
 				type: "advancement"
-				title: "Find a &7Clay Island&r"
+				title: "找到一座&7粘土岛&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				title: "找到一座&7黏土岛&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 9]
```diff
@@ -31,12 +31,12 @@
 			}]
 		}
 		{
-			title: "&7Clay&r, &#C2B280Sand&r \\& &#909BA3Gravel&r for &7Grout&r"
+			title: "为了水泥去搞到黏土沙子和砂砾"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "去搞些&7黏土&r、&#C2B280沙子&r和&#909BA3砂砾&r来制作&7砖泥&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 10]
```diff
@@ -31,12 +31,12 @@
...
 			icon: "tconstruct:grout"
 			x: -5.0d
 			y: -8.0d
-			subtitle: "Making Grout"
-			description: ["As you travel the world, those materials will really be necessary."]
+			subtitle: "制造水泥"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			subtitle: "制造砖泥"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 11]
```diff
@@ -47,14 +47,14 @@
 				{
 					id: "69D93E0629AF5C10"
 					type: "item"
-					title: "16x &7Clay&r"
+					title: "16x&7粘土&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "16x&7黏土&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 12]
```diff
@@ -81,12 +81,12 @@
 			}]
 		}
 		{
-			title: "&2Tinkers' Construct Slime Islands&r"
+			title: "&2匠魂史莱姆空岛&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "&2匠魂黏液空岛&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 13]
```diff
@@ -81,12 +81,12 @@
...
 			icon: "tconstruct:earth_earth_slime_grass"
 			disable_toast: true
 			x: -3.0d
 			y: -10.0d
-			description: ["Find one of the four &2Tinkers' Construct Slime Islands&r."]
+			description: ["找到四种&2匠魂史莱姆空岛&r中的一种。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			description: ["找到四种&2匠魂黏液空岛&r中的一种。"]
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 14]
```diff
@@ -95,7 +95,7 @@
 				{
 					id: "2F17AC75EE4BF554"
 					type: "advancement"
-					title: "Find an &2Earthslime Island&r"
+					title: "找到一座&2大地史莱姆岛&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "找到一座&2大地黏液岛&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 15]
```diff
@@ -104,7 +104,7 @@
 				{
 					id: "28115D66CAB7B751"
 					type: "advancement"
-					title: "Find a &3Skyslime Island&r"
+					title: "找到一座&3碧空史莱姆岛&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "找到一座&3碧空黏液岛&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 16]
```diff
@@ -122,7 +122,7 @@
 				{
 					id: "3A6765962EB5CC2B"
 					type: "advancement"
-					title: "Find an &5Enderslime Island&r"
+					title: "找到一座&5末影史莱姆岛&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "找到一座&5末影黏液岛&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 17]
```diff
@@ -131,7 +131,7 @@
 				{
 					id: "10ABC481802465E1"
 					type: "checkmark"
-					title: "Find &2Tinkers' Construct Slime Island&r"
+					title: "找到&2匠魂史莱姆空岛&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "找到&2匠魂黏液岛&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 18]
```diff
@@ -181,7 +181,7 @@
 			]
 		}
 		{
-			title: "&2Slime&r"
+			title: "&2史莱姆&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "&2黏液&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 19]
```diff
@@ -191,7 +191,7 @@
 				{
 					id: "4CF6F7DF3A32CEA2"
 					type: "item"
-					title: "Congealed Slime"
+					title: "凝固粘液块"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "凝固黏液块"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 20]
```diff
@@ -203,7 +203,7 @@
 				{
 					id: "41A75385842C8BAA"
 					type: "item"
-					title: "Slime Dirt"
+					title: "粘液泥土"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "黏液泥土"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 21]
```diff
@@ -232,7 +232,7 @@
 				{
 					id: "5A0D685D56365735"
 					type: "item"
-					title: "Slimy Logs"
+					title: "黏滑原木"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "黏液原木"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 22]
```diff
@@ -244,7 +244,7 @@
 				{
 					id: "070FCF56ED24BF9A"
 					type: "item"
-					title: "Slimy Saplings"
+					title: "黏滑树苗"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "黏液树苗"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 23]
```diff
@@ -262,15 +262,16 @@
 			}]
 		}
 		{
+			title: "史莱姆弹弓"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "黏液弹弓"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 24]
```diff
@@ -262,15 +262,16 @@
...
 			y: -8.0d
-			subtitle: "Making Travelling Cool"
+			subtitle: "让旅行更酷"
 			dependencies: ["4866C5C9EF6794DA"]
 			id: "7A172669A4672D91"
 			tasks: [{
 				id: "6C75B5590C3D79B3"
 				type: "item"
-				title: "Slime Sling"
+				title: "史莱姆弹弓"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				title: "黏液弹弓"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt (hunk 1 above)
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				title: "3x工匠青铜块"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 25]
```diff
@@ -366,22 +367,22 @@
...
-				"Your smeltery has some requirements to run correctly. "
+				"为了能够正确的运转，你的冶炼炉需要满足一些要求。"
 				""
-				"First of all, your smeltery must have a floor made of acceptable smeltery blocks excluding glass. It also has to be rectangular, it can't be L shaped, circle shaped or dodecahedron shaped."
-				"The max size of your smeltery is 16x16x65 external or 14x14x64 internal."
+				"首先，你的冶炼炉必须有一个由玻璃以外的冶炼炉方块铺成的地板。它必须是矩形的，不能是L形，圆形或是十二边形。"
+				"冶炼炉的最大尺寸是16x16x65，其有着14x14x64的内部空间。"
 				""
-				"&6&l&nAlloying :&r"
+				"&6&l&n合金化：&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"&6&l&n合金：&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 26]
```diff
@@ -366,22 +367,22 @@
...
 				""
-				"Alloying is a process that will allow you to combine your materials into stronger ones."
+				"合金化是一个允许你将材料组合成更强大形式的过程。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"合金是一个允许你将材料组合成更强大形式的过程。"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 27]
```diff
@@ -366,22 +367,22 @@
...
 				""
-				"In order to alloy, you will have to find two materials that can be alloyed together. "
+				"为了制造合金，你需要寻找两种不同的可以进行合金化的材料。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"为了制造合金，你需要寻找两种不同的可以进行合金的材料。"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt (hunk 2 above)
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
					title: "2x浇注口"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 28]
```diff
@@ -437,17 +438,17 @@
 			}]
 		}
 		{
-			title: "Seared Ingot \\& Fuel Gauge"
+			title: "材料量器和燃料量器"
 			x: -7.5d
 			y: -5.5d
 			description: [
-				"Seared Ingot Gauge is ideal for metals. It can hold 32 ingots of fluid (4.6 buckets)."
+				"焦黑材料量器是盛放材料的理想容器，它能够存放32锭的液体（4.6b）"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"焦黑材料量器是盛放材料的理想容器，它能够存放32锭的液体（4.6B）"
```
匠魂规范，这里B翻译成中文需大写

File: config/ftbquests/quests/chapters/chaptre_2.snbt (hunk 3 above)
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"从根本上说，焦黑燃料量器和焦黑材料量器相同。它可以作为冶炼炉的心脏，并且能够存放4桶燃料。"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 29]
```diff
@@ -477,17 +478,17 @@
...
-			subtitle: "Early Smelting"
+			subtitle: "早期的冶炼炉"
 			description: [
 				"{image:chroma:textures/quests/melter.png width:200 height:150 align:1}"
 				""
-				"&l&n&6Operation :&r"
+				"&l&n&6操作方法：&r"
 				""
-				"There are as you can see &33&r slots where you put in what you want to melt. It may be ores, ingots, or anything else you feel like attempting to melt!."
+				"你可以看见&3三个&r格子来放置你想融化的东西。它可以是矿石，锭或是其他你觉得可以融化的东西！"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"你可以看见&3三个&r格子来放置你想融化的东西。它可以是矿石，锭或是其他你觉得可以熔化的东西！"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 30]
```diff
@@ -477,17 +478,17 @@
...
 				""
-				"The &3Gauge&r in the middle is where your molten material will show. Remember that you may only have one molten material at a time in the melter."
+				"你已经融化了材料将会展示在&3中间的槽&r里。记住，同一时间你只能有一种融化的材料在这座融化炉里。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"你已经熔化了材料将会展示在中间的&3槽&r里。请记住，焦黑熔化炉中一次只有一种熔融材料。"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 31]
```diff
@@ -533,12 +534,12 @@
...
 		}
 		{
-			title: "Seared Bricks"
+			title: "焦黑砖"
 			x: -7.5d
 			y: -8.0d
 			shape: "gear"
-			subtitle: "Smelting Grout"
-			description: ["&7&lTinker Construct's&r best friend"]
+			subtitle: "熔炼水泥"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			subtitle: "熔炼砖泥"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 32]
```diff
@@ -819,7 +820,7 @@
 			}]
 		}
 		{
-			title: "&dPiggybackpack&r"
+			title: "&d肩托架&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "&d肩驮架&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 33]
```diff
@@ -852,18 +853,18 @@
...
 		{
-			title: "Gold Cast"
+			title: "铸模"
 			icon: "tconstruct:blank_cast"
 			x: -7.5d
 			y: 2.5d
-			subtitle: "Reusable "
-			description: ["&e&l&nGold Casts&r are reusable&r. To make one, simply pour one ingot worht of molten gold over a tool part placed in a casting table. The tool part is used up in the process."]
+			subtitle: "重复利用"
+			description: ["&e&l&n金质铸模&r是可重复利用的&r。想要制作一个时，只需要简单地将想要铸造的部件放置在焦黑铸件台上，在倒一锭量的融熔金在上面即可。在这个过程中，该部件会被消耗。"]
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			description: ["&e&l&n金质铸模&r是可重复利用的&r。想要制作一个时，只需要简单地将想要铸造的部件放置在焦黑铸件台上，再倒1锭量的融熔金在上面即可。在这个过程中，该部件会被消耗。"]
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt (hunk 4 above)
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "燧石和 _____"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 34]
```diff
@@ -953,7 +954,7 @@
 			}]
 		}
 		{
-			title: "&5The Nether&r"
+			title: "&5地狱&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "&5下界&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 35]
```diff
@@ -992,7 +993,7 @@
 			}]
 		}
 		{
-			title: "&#61666ACobalt&r"
+			title: "&#61666A铅&r"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			title: "&#61666A钴&r"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 36]
```diff
@@ -1085,23 +1086,23 @@
 			x: -4.75d
 			y: 13.0d
 			shape: "square"
-			subtitle: "Scorched Alloying "
+			subtitle: "焦褐合金"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			subtitle: "焦褐合金炉"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt (hunk 5 above)
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"从根本上说，焦褐燃料量器和焦褐材料量器相同，都可以作为焦褐燃料容器。它可以被用作为熔铸炉燃料容器，并且能够存放4桶燃料。"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 37]
```diff
@@ -1142,17 +1143,16 @@
 			}]
 		}
 		{
-			title: "Scorched Ingot \\& Fuel Gauge"
 			x: -4.75d
 			y: 10.75d
 			description: [
-				"Scorched Ingot Gauge is ideal for metals. It can hold 32 ingots of fluid (4.6 buckets)."
+				"焦褐材料量器是盛放材料的理想容器，它能够存放32锭的液体（4.6b）。"
```
Comment (Cactusstudent) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"焦褐材料量器是盛放材料的理想容器，它能够存放32锭的液体（4.6B）。"
```
理由同上

**Review by Cactusstudent — CHANGES_REQUESTED** [review]
……终于审核完了，累死

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 38]
```diff
@@ -1413,23 +1413,23 @@
...
+				"&n&6&l结构要求：&r"
 				""
-				"Your foundry has some requirements to run correctly. "
+				"为了能够正确地运转，你的熔铸炉需要满足一些要求。"
 				""
-				"First of all, your foundry must have a floor made of acceptable foundry blocks excluding glass. It also has to be rectangular, it can't be L shaped, circle shaped or dodecahedron shaped."
+				"首先，你的熔铸炉必须有一个由玻璃以外的熔铸炉方块铺成的地板。它也必须是矩形的，不能是L形，圆形或是十二边形。"
 				""
-				"It must have corners and edges to work correctly ! "
+				"它必须要有全部的边和角以正确运转！"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
				"它必须要有全部的边和角以正确生效！"
```

File: config/ftbquests/quests/chapters/chaptre_2.snbt [hunk 39]
```diff
@@ -1574,8 +1574,8 @@
 			x: -11.0d
 			y: 8.75d
 			shape: "hexagon"
-			subtitle: "AllTheModium Harvest Level required"
-			description: ["&aWarped Forests and Crimson Forest, Y 107 to 113"]
+			subtitle: "ATM采掘等级需求"
```
Comment (Ricky-fight) [inline config/ftbquests/quests/chapters/chaptre_2.snbt]:
```suggestion
			subtitle: "ATM金属采掘等级需求"
```

**Review by Ricky-fight — APPROVED** [review]

**Review by AnzhiZhang — APPROVED** [review]

**Pull request merged**